hummingbot/client/ui/custom_widgets.py

```diff
@@ -45,6 +38,48 @@ def validate_and_handle(self):
                 self.reset()
 
 
+class FormattedTextLexer(Lexer):
+
+    PROMPT_TEXT = ">>> "
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.html_tag_css_style_map: Dict[str, str] = {style: css for style, css in load_style().style_rules}
+
+        # Maps specific text to its corresponding UI styles
+        self.text_style_tag_map: Dict[str, str] = text_ui_style
+
+    def get_css_style(self, tag: str) -> str:
+        return self.html_tag_css_style_map.get(tag, "")
+
+    def lex_document(self, document: Document) -> Callable[[int], StyleAndTextTuples]:
+        lines = document.lines
+
+        def get_line(lineno: int) -> StyleAndTextTuples:
+            "Return the tokens for the given line."
+            try:
+                current_line = lines[lineno]
+                if len(current_line) == 0:
+                    return [("", current_line)]
+
+                # Command prompt
+                if current_line.startswith(self.PROMPT_TEXT):
```

Comment:
Maybe you can check also against a list of possible commands and so that the string ends right after that, to further eliminate mismatches. If a user types ">>>" it will be marked in the output as a command.